src/components/organisms/PluginManager/PluginInstallModal.tsx

```diff
@@ -85,12 +128,22 @@ function PluginInstallModal(props: {isVisible: boolean; onClose: () => void}) {
           See Plugin&apos;s documentation for more information.
         </Button>
       </p>
-      <Input placeholder="Enter Plugin URL" defaultValue={pluginUrl} onChange={e => setPluginUrl(e.target.value)} />
-      {errorMessage && (
-        <div>
-          <p>{errorMessage}</p>
-        </div>
-      )}
+      <Form layout="vertical" form={pluginForm} onFinish={onFinish} initialValues={() => formValues}>
+        <Form.Item
+          name="pluginUrl"
+          label="URL"
+          required
+          tooltip={PLUGIN_URL_INPUT_TOOLTIP}
+          rules={[
+            {
+              required: true,
+              message: 'Please provide your plugin URL!',
+            },
+          ]}
+        >
+          <Input ref={inputRef} placeholder="Enter Plugin URL" />
+        </Form.Item>
```

Comment:
I still think we should have some kind of info on what kind of url(s) we support... @chlete what do you think?